### DISCUSSION TITLE
Small error in french translation

### DISCUSSION THREAD

**Discussion by Npa972** [audacious-media-player/audacious#650] [General]
In the track context menu "Paste at end" is translate by "Copier à la fin" wich leads to confusion.
Paste at end = Coller à la fin

Sorry I'm sure it is not the good place but i couldn't open a issue.

**Comment by radioactiveman** [comment]
That's fine, we are currently migrating to GitHub Issues. Until then you can report such things here.

I have contacted the translator for French on [Transifex](https://www.transifex.com/) about it, thanks for reporting.

**Comment by AO-LocLab** [comment]
Thank you @Npa972 for reporting this error.
It will be fixed with the next Audacious release.
